src/state-summary/state-card-climate.html

```diff
@@ -43,8 +43,7 @@
             [[computeTargetTemperature(stateObj)]]
           </span>
         </div>
-
-        <div class='current'>
+        <div class='current' hidden$='[[!hasCurrentTemperature(stateObj)]]'>
```

Comment:
No reason to add a function. You can just pass in `stateObj.attributes.current_temperature`. If the temperature cannot be rendered, we should still show it.